### PR TITLE
Syntax, language, consistency

### PR DESCRIPTION
**Pull request by AJFaraday**
First things first, loving the style guide, it was a really useful read. Thanks for putting this together.

This pull request is mostly a cople of style changes, inconsistent language, extra brackets etc.

There's two parts which have an editorial decision to be made...

* In defining scopes as a class method, I've added a note that these can not be added in the same way as named scopes...

```ruby
class User < ActiveRecord::Base

  def User.old
    where('age > ?', 80)
  end

  def User.heavy
    where('weight > ?', 200)
  end

end 
```

then in usage

```
> User.old.heavy
NoMethodError: undefined method `heavy' for #<ActiveRecord::Relation:0x0000000acfabe8>
```

* I've added a slightly clearer example for sexy validations, showing how you can convert multiple method calls to multiple arguments to the `validates` method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -367,9 +367,10 @@ programming resources.
   ```Ruby
   # bad
   validates_presence_of :email
-
+  validates_length_of :email, maximum: 100 
+ 
   # good
-  validates :email, presence: true
+  validates :email, presence: true, length: {maximum: 100}
```
Comment (bbatsov) [inline README.md]: Add spaces inside the hash.

File: README.md [hunk 2]
```diff
@@ -422,6 +423,7 @@ programming resources.
   complicated, it is preferable to make a class method instead which serves the
   same purpose of the named scope and returns an `ActiveRecord::Relation`
   object. Arguably you can define even simpler scopes like this.
+  Note: this style of scoping can not be chained in the same way as named scopes.
```
Comment (bbatsov) [inline README.md]: I'd add this note after the code example with another code example.

**Comment by bbatsov** [comment]
I see a few commits in this PR that can probably be removed (something was added and removed afterwards).

**Comment by AJFaraday** [comment]
Yes, there was an area which was changed and then reverted, is this a problem if the finishing state is okay? What can I do to ammend this?

**Comment by bbatsov** [comment]
@AJFaraday You can squash those commits with some other commits. Otherwise there're just noise in the history.

File: README.md [hunk 3]
```diff
@@ -422,6 +423,7 @@ programming resources.
   complicated, it is preferable to make a class method instead which serves the
   same purpose of the named scope and returns an `ActiveRecord::Relation`
   object. Arguably you can define even simpler scopes like this.
+
```
Comment (bbatsov) [inline README.md]: This blank line is redundant.

File: README.md [hunk 4]
```diff
@@ -431,6 +433,34 @@ programming resources.
     end
   end
   ```
+  
+  Note: this style of scoping can not be chained in the same way as named scopes. For instance:
```
Comment (bbatsov) [inline README.md]: You can rephrase this as "Note that...".

File: README.md [hunk 5]
```diff
@@ -431,6 +433,34 @@ programming resources.
...
+      where('age > ?', 80)
+    end
+
+    def User.heavy
+      where('weight > ?', 200)
+    end
+
+  end 
+  ```
+  In this style both old and heavy work individually, but you can not call `User.old.heavy`, to chain these scopes use:
```
Comment (bbatsov) [inline README.md]: Add blank lines around this and put old and heavy in backticks.

File: README.md [hunk 6]
```diff
@@ -431,6 +433,34 @@ programming resources.
     end
   end
   ```
+  
+  Note: this style of scoping can not be chained in the same way as named scopes. For instance:
+  
+  ```Ruby
+  # unchainable
+  class User < ActiveRecord::Base
+
```
Comment (bbatsov) [inline README.md]: Remove the blank lines around the class body.

File: README.md [hunk 7]
```diff
@@ -431,6 +433,34 @@ programming resources.
...
+      where('weight > ?', 200)
+    end
+
+  end 
+  ```
+  In this style both old and heavy work individually, but you can not call `User.old.heavy`, to chain these scopes use:
+  ```Ruby
+  # chainable
+  class User < ActiveRecord::Base
+
```
Comment (bbatsov) [inline README.md]: Here as well.

**Comment by AJFaraday** [comment]
I've made my first attempt here at a git rebase to remove the two extra commits, and now I'm not sure if I've done something wrong, but it looks like there's more commits.

**Comment by AJFaraday** [comment]
Looking at the actual repo, it looks like the rebase worked, and the history on the pull request is showing them multiple times. I've made the requested changes.

**Comment by bbatsov** [comment]
Something seems wrong to me - the commits exist several times with different committers. Maybe it'd be easier for you to squash them in a single commit with a longer description.
